Mark BetterMountHUD as conflict

MylesGit opened the pull request:
BetterMountHUD is Fabric only so this resolves issue #15 

No use in having duplicate behavior. Might want to mention this in the description on all mod pages.

MylesGit committed:
Mark BetterMountHUD as conflict

Fyoncle commented:
@MylesGit Also add [HorseInBoat](https://modrinth.com/mod/horseinboat) as conflict please.

MylesGit committed:
Add HorseInBoat as conflict

Fyoncle commented:
Thanks man!

MylesGit committed:
Add HorseBuff as conflict

HorseBuff is now integrated into HorseMan. Hence current and future versions of HorseMan should not support HorseBuff.

Fyoncle commented:
I forgot about that, lol

MylesGit commented:
These are the current mods I can think of that will directly conflict with HorseMan's behavior (and cause issues). Let me know if I missed anything.

Fyoncle commented:
> These are the current mods I can think of that will directly conflict with HorseMan's behavior (and cause issues). Let me know if I missed anything.

[Swimming Horses](https://modrinth.com/mod/swimming-horses) it is, better than swimming mechanic of HorseMan but 🤷

Fyoncle commented:
Tbh since its configurable and some people might prefer style of Swimming Horses, it might be better not to add this one

MylesGit commented:
They do overlap features but I doubt you would experience any crashes or bugs by having both installed. It's best practice to leave mods like this alone.

mortuusars commented:
@Fyoncle I haven't been able to run Horse In Boat with Horseman (at least in my dev env) because it requires 1.21 mc version, not 1.21.1. But there shouldn't be any incompatibility with it, you can disable Horseman's fit in boats features in config and it should run fine.

Fyoncle commented:
> @Fyoncle I haven't been able to run Horse In Boat with Horseman (at least in my dev env) because it requires 1.21 mc version, not 1.21.1. But there shouldn't be any incompatibility with it, you can disable Horseman's fit in boats features in config and it should run fine.

I was talking about 1.20.1 but you're right!

mortuusars commented:
As for other mods - I'll try to make my mixins more compatible, so it doesn't crash. And if it will work out, I'd rather not make these mods "incompatible" (except Horse Buff maybe). 
Since all Horseman features can be turned off in config - it should cause issues when turned off.

MylesGit commented:
I can remove everything but HorseBuff if needed. But, be aware that BetterMountHUD does conflict atm in case someone else opens an issue.

mortuusars commented:
I've merged the PR as is. Will remove the mods myself.
Also crash with Better Mount HUD will be fixed in next release.

Fyoncle commented:
> I've merged the PR as is. Will remove the mods myself. Also crash with Better Mount HUD will be fixed in next release.

Will next release also fix the incompatibility and feature request I opened? Very excited to add this mod to my modpack.

mortuusars commented:
Probably. I have not reached those issues yet.

Fyoncle commented:
> Probably. I have not reached those issues yet.

Ah gotcha, good luck on development! I really like the mod.
